src/components/Select/index.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Select from "./index";
+
+describe("Select", () => {
+  it("renders the default option when no defaultValue is set", () => {
+    render(
+      <Select
+        prompt="Choose an option"
+        options={[
+          { id: 1, name: "First Option" },
+          { id: 2, name: "Second Option" },
+        ]}
+      />
+    );
+
+    expect(screen.getByText("Choose an option")).toBeVisible();
+  });
+
+  it("renders the default option when defaultValue is null", () => {
+    render(
+      <Select
+        defaultValue={null}
+        prompt="Choose an option"
+        options={[
+          { id: 1, name: "First Option" },
+          { id: 2, name: "Second Option" },
+        ]}
+      />
+    );
+
+    expect(screen.getByText("Choose an option")).toBeVisible();
+  });
+
+  it("renders the the correct option when the defaultValue is set", () => {
```

Comment:
What is the "correct option"? Maybe something along the lines of:

```suggestion
  it("renders the default option as the option with the matching id as the defaultValue", () => {
```